layouts/shortcodes/pages/newsletter.html

```diff
@@ -1,3 +1,50 @@
+<div class="panel-group margin-top-30" id="accordion" role="tablist" aria-multiselectable="true">
+  <div class="panel panel-default">
+    <div class="panel-heading" role="tab" id="heading2019">
```

Comment:
ID's and anchors should be updated to reflect 2020 section rather than 2019

Reply:
Done! thx